tools/apiview/parsers/apiview-treestyle-parser-schema/model.tsp

```diff
@@ -0,0 +1,94 @@
+import "@typespec/json-schema";
+
+using TypeSpec.JsonSchema;
+
+@jsonSchema
+
+/** ReviewFile represents entire API review object. This will be processed to render review lines. */
+model CodeFile {
+  PackageName: string;
+  PackageVersion: string;
+  /** version of the APIview language parser used to create token file*/
+  ParserVersion: string;
+  Language: "C"|"C++"|"C#"|"Go"|"Java"|"JavaScript"|"Kotlin"|"Python"|"Swagger"|"Swift"|"TypeSpec";
```

Comment:
For my own curiosity and my lack of knowledge here 😨 , we support Java with variant Android, and also Kotlin? do people use both? or how does it work?

Reply:
It's handled by Java parser.